specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2023-02-15-preview/share.json

```diff
@@ -0,0 +1,2019 @@
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/PurviewShareError"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ReceivedShares_ActivateTenantEmailRegistration": {
+            "$ref": "./examples/ReceivedShares_ActivateTenantEmailRegistration.json"
+          }
+        }
+      }
+    },
+    "/registerEmail": {
+      "post": {
+        "tags": [
+          "TenantEmailRegistration"
+        ],
+        "summary": "Registers the tenant and email combination for activation.",
+        "description": "Register an email for the current tenant",
+        "operationId": "ReceivedShares_RegisterTenantEmailRegistration",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "header",
+            "name": "repeatability-request-id",
+            "description": "If specified, the client directs that the request is repeatable; that is, that the client can make the request multiple times with the same Repeatability-Request-Id and get back an appropriate response without the server executing the request multiple times. The value of the Repeatability-Request-Id is an opaque string representing a client-generated, globally unique for all time, identifier for the request. It is recommended to use version 4 (random) UUIDs.",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/TenantEmailRegistration"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/PurviewShareError"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ReceivedShares_RegisterTenantEmailRegistration": {
+            "$ref": "./examples/ReceivedShares_RegisterTenantEmailRegistration.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AdlsGen2AccountSink": {
+      "description": "AdlsGen2 Sink",
+      "required": [
+        "properties",
+        "storeKind",
+        "storeReference"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Sink"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AdlsGen2AccountSinkProperties",
+          "x-ms-client-flatten": true
```

Comment:
Please remove all instances of "x-ms-client-flatten". This extension is deprecated and no longer supported by our client generators.